dwave/preprocessing/include/dwave-preprocessing/implication_network.hpp

```diff
@@ -116,7 +117,7 @@ class stronglyConnectedComponentsInfo {
   stronglyConnectedComponentsInfo(int num_components,
                                   std::vector<int> vertex_to_component_map,
                                   mapper_t &mapper)
-      : num_components(num_components), num_vertices(mapper.num_vertices()),
+      : num_vertices(mapper.num_vertices()), num_components(num_components), 
```

Comment:
I am fine with cleaning up some of the warnings, though a bigger refactor of the roof duality stuff is needed. But ideally that would be in a separate PR.

Reply:
Yes.  I gave up on fixing the warnings: there were just too many.  Instead, I suppressed the most egregious offenders to mitigate some of the noise